.github/scripts/tkn_check_task_runasuser.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
+
+#
+# tkn_check_task_runasuser.sh
+#
+# Scans Tekton tasks in the CHANGED_FILES environment variable and ensures that Tekton tasks using Trusted Artifacts
+# (those declaring ociStorage parameter) with non-konflux images have proper security context (runAsUser: 1001) in their stepTemplate.
+#
+
+set -euo pipefail
+
+# Function to log messages to stderr
+log() {
+    echo "$@" >&2
+}
+
+# Function to display usage
+usage() {
+    log "Usage: CHANGED_FILES=\"file1.yaml file2.yaml\" $0"
+    log ""
+    log "  CHANGED_FILES: Space-delimited list of files to check"
+    log ""
+    log "This script checks that any Tekton task using Trusted Artifacts (declaring ociStorage parameter)"
+    log "with non-konflux images has 'runAsUser: 1001' defined in its stepTemplate.securityContext"
```

Comment:
What does it mean "tasks using TA with non-konflux images"? How is create-advisory using a non-konflux image? Isn't the utils image a konflux image? Can you clarify this?

Reply:
See https://github.com/scoheb/release-service-catalog/blob/108ed366a0db8f31cafbbd08d5076ff867df5cca/tasks/managed/create-advisory/create-advisory.yaml#L160

Reply:
updated wording to use "non-release-service-utils" image